distribution/lib/Standard/Base/0.0.0-dev/src/Data/Ordering.enso

```diff
@@ -25,9 +19,24 @@ from_sign sign = if sign == 0 then Equal else
    The result should be returned in terms of how `this` orders in comparison to
    `that`. So, if `this` is greater than `that`, you should return `Greater.`
 type Ordering
-    Less
-    Equal
-    Greater
+
+    ## A representation of the relative ordering between two values.
+
+       These values must be able to be ordered with respect to each other.
+    @Builtin_Type
+    type Ordering
+
+    ## A representation that the first value orders as less than the second.
+    @Builtin_Type
+    type Less
+
```

Comment:
Why is `Ordering` a separate type under the `Ordering` umbrella? Is it to make static methods work? Is this temporary?

It looks weird that the, meant-to-be-supertype `Ordering` is placed at the same level as its meant-to-be subtypes (`Less` etc.)